server/app/authorize_station.py

```diff
@@ -133,9 +134,11 @@ def api(station_id: str):
     '''
     @wraps(f)
     def decorated_function(*args, **kws):
+        logging.info("Verifying request.")
         err, id_ = _verify_request()
         if err is not None:
             if app.config["security"].get("ignore_hmac_validation_errors", "false").lower() != "true":
+                logging.warn("Authorization failed %s" % err)
```

Comment:
I think that it should be the "debug" level.
As this part of the code is in the "non-trusted" area hackers may send many invalid requests and log size may exceed free space. This logging point should be used only on-demand to diagnose problems.

Reply:
Decreased logging level. There was a `Verifying request` log that I lowered to debug. With the authorization failed, I think this shouldn't require turning on the debugging level. There is something wrong happening in the system and the admin should be notified. But I understand the concern about the code being too verbose. So I've turned this down to info. Would that be a reasonable compromise?